packages/demo/src/index.js

```diff
@@ -102,7 +105,7 @@ async function main() {
     await playerAnalytics.init({
       sessionId: `demo-page-${Date.now()}`,
       live: false,
-      contentId: "BBB",
+      contentId: 'BBB',
```

Comment:
This should probably be something more relevant.

Reply:
How about the manifest URL?

Reply:
Or the file extension part of it?

Reply:
I'm thinking (for example) "mpd-1644915348006" (Date.now())

Reply:
Whatever we choose it should be associated to the content. Meaning that playback of the same content will get the same ID. I suggest in this case to have the manifest URL as the contentId.